docs/_data/mesheryctlcommands/apps/view/flags/view-all.yaml

```diff
@@ -0,0 +1,4 @@
+name: --all, -a
+description: show all appliaction file content
+usage: |
+    mesheryctl app view [appliaction-name|appliaction-id] --all
```

Comment:
@afzal442 I can see PR is in DRAFT, but still want you to know that usage for `view all` should be like that : 
```
mesheryctl app view  --all
```
Otherwise will give below mentioned error
```
-a cannot be used when [application-name|application-id] is specified
```

Reply:
`mesheryctl app view -a` and `mesheryctl app view -all` both will work
I was just referring to usage mentioned here.
`mesheryctl app view app_name --all` will not work
and usage should be `mesheryctl app view -a` or `mesheryctl app view -all`

Reply:
thanks, this makes sense now.